blob/azureblob/example_test.go

```diff
@@ -42,22 +42,33 @@ func Example() {
 	// Creates a *blob.Bucket backed by Azure Storage Account
 	b, _ := azureblob.OpenBucket(ctx, serviceURL, bucketName, &azureOpts)
 
-	// Obtain the native blob attributes
```

Comment:
Using the `As` functionality should be rare for users (hopefully). Could you split this into two examples, one which focuses on creating the bucket and doing a read (similar to `gcsblob/example_test.go`), and another one with all of the `As` stuff? Note that you can have `// Output` for the former at least to make sure it runs.

Reply:
Done.